1_portfolio_check_initialisation.R

```diff
@@ -30,7 +30,7 @@ source("0_portfolio_input_check_functions.R")
 source("0_global_functions.R")
 source("0_sda_approach.R")
 
-project_name <- "Laura_test_project_2019_jk"
+project_name <- "liechtenstein_p2020"
```

Comment:
While we're here, I propose we extract the "project_name" to a config file. 
This way we won't have merge conflicts every time somebody is working on a different project

Reply:
That sounds like a good plan. The project_name and the file paths below would then be in a config file in the repo I guess.. Can you implement this?

Reply:
Moved to an issue.